src/commissaire/bus/__init__.py

```diff
@@ -99,14 +100,18 @@ def request(self, routing_key, method, params={}, **kwargs):
         result = response_queue.get(block=True, timeout=10)
         result.ack()
 
-        if 'error' in result.payload.keys():
+        payload = result.payload
+        if isinstance(payload, str):
+            payload = json.loads(payload)
+
+        if 'error' in payload.keys():
             self.logger.warn(
                 'Error returned from the message id "{}"'.format(
```

Comment:
Missed this before, but the string template needs another placeholder for the payload.